src/DacpacTool/DacpacTool.csproj

```diff
@@ -13,7 +13,7 @@
 
   <ItemGroup>
     <!-- These packages contain DacFX analysis rules, which must be located with the executable in order to be discovered-->
-    <PackageReference Include="ErikEJ.DacFX.SqlServer.Rules" Version="1.0.0-preview8" IncludeAssets="runtime" />
```

Comment:
I feel like this approach of continually incrementing the preview version doesn't scale, especially since we're just rubber-stamping each increment anyway. Could we move to using a wildcard, or possibly a maximum version constraint instead?

Reply:
@jeffrosenberg Sure, what would that look like?

Reply:
1.0.* ?

Reply:
Or I could release 1.0.0 and we could add the packages to dependabot?

Reply:
Sorry I didn't respond to this, you had already chosen a path and merged a solution by the time I got to it :slightly_smiling_face: 

In case you haven't seen it, here's the [NuGet Package Version Reference](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort). My recommendation would be to use the "maximum version, exclusive" format (i.e. `(,1.0)`) to set an upper bound at either `1.1.0` or `2.0.0`, and then you're free to release fixes to your package within that range.